lib/cucumber-api/steps.rb

```diff
@@ -27,10 +27,39 @@
   end
 end
 
+Then (/^I save the "([^"]+)"$/) do |k|
+  if @response.nil?
+    raise 'No response found.'
+  end
+
+  if k[0] == '$'
+    v = k
+  else
+    v = "$.#{k}"
+  end
+
+  k.gsub!(/[^0-9a-zA-Z_]/, '')
+  instance_variable_set("@#{k}", @response.get(v))
+end
+
+Then (/^I save the "([^"]+)" as "([^"]+)"$/) do |k, v|
+  if @response.nil?
+    raise 'No response found.'
+  end
+
+  k = "$.#{k}" unless k[0] == '$'
+  instance_variable_set("@#{v}", @response.get(k))
+  puts 
+end
+
 When(/^I set JSON request body to '(.*?)'$/) do |body|
   @body = JSON.parse body
 end
 
+When(/^I set JSON request body to:$/) do |body|
```

Comment:
That's one great thing about writing code, there are always new things to learn ;)